airsonic-main/src/main/java/org/airsonic/player/spring/LoggingExceptionResolver.java

```diff
@@ -4,12 +4,14 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.core.Ordered;
+import org.springframework.stereotype.Component;
 import org.springframework.web.servlet.HandlerExceptionResolver;
 import org.springframework.web.servlet.ModelAndView;
 
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+@Component
```

Comment:
I would maybe give this an Order ?
(even though it implements Ordered)

Reply:
Also out of scope. This PR is soley about removing the XML files and keeping the beans as close as possible to how they were defined before.